client/src/components/grants/Details.tsx

```diff
@@ -20,16 +22,52 @@ export default function Details({
   bannerImg,
   logoImg,
   preview,
+  alert,
 }: {
   project?: Metadata | FormInputs | Project;
   updatedAt: string;
   createdAt: string;
   bannerImg: string | Blob;
   logoImg: string | Blob;
   preview?: boolean;
+  alert?: boolean | undefined;
 }) {
+  const [showAlert, setShowAlert] = useState(alert ?? true);
+  const discordLink = (
+    <a className="text-purple-600" href="https://discord.gg/nwYzGuuruJ">
+      Grant Hub Discord
+    </a>
+  );
+
+  const successfullSubmissionBody = (
+    <p className="text-black">
+      Your application will be reviewed by the grant round team and you will
+      receive an email if your project is approved for the grant round. If you
```

Comment:
This makes me think I will not hear anything back if it's not approved - is that the case, or should we change the copy a little?

@michellema1208 @willsputra

Reply:
This depends on the round operators I think, not sure if they'll send an email to everyone or just the approved ones. What do you think @michellema1208 ?

Reply:
Hey @DanieleSalatti @willsputra - yeah let's make this message a bit more generic for now - maybe let's just have the copy say "Thanks for applying to <Program> <Round>! Your application has been received, and the <Program> team will review and reach out with next steps." 

I think showing MVP application status will be a next good-to-have feature we can prioritize, pending completion of priority tickets.

Reply:
I like this approach @michellema1208 I can update to show  "Thanks for applying to ! Your application has been received, and the team will review and reach out with next steps."

Reply:
@michellema1208 going to move adding the alert message to another PR to use @gravityblast new component. Just let me know on the copy for now.